crates/sound/src/audio_backends/pipewire.rs

```diff
@@ -1,30 +1,542 @@
+                }
+            }
+
+            let info = spa_audio_info_raw {
+                format: match params.format {
+                    VIRTIO_SND_PCM_FMT_MU_LAW => SPA_AUDIO_FORMAT_ULAW,
+                    VIRTIO_SND_PCM_FMT_A_LAW => SPA_AUDIO_FORMAT_ALAW,
+                    VIRTIO_SND_PCM_FMT_S8 => SPA_AUDIO_FORMAT_S8,
+                    VIRTIO_SND_PCM_FMT_U8 => SPA_AUDIO_FORMAT_U8,
+                    VIRTIO_SND_PCM_FMT_S16 => SPA_AUDIO_FORMAT_S16,
+                    VIRTIO_SND_PCM_FMT_U16 => SPA_AUDIO_FORMAT_U16,
+                    VIRTIO_SND_PCM_FMT_S18_3 => SPA_AUDIO_FORMAT_S18_LE,
+                    VIRTIO_SND_PCM_FMT_U18_3 => SPA_AUDIO_FORMAT_U18_LE,
+                    VIRTIO_SND_PCM_FMT_S20_3 => SPA_AUDIO_FORMAT_S20_LE,
+                    VIRTIO_SND_PCM_FMT_U20_3 => SPA_AUDIO_FORMAT_U20_LE,
+                    VIRTIO_SND_PCM_FMT_S24_3 => SPA_AUDIO_FORMAT_S24_LE,
+                    VIRTIO_SND_PCM_FMT_U24_3 => SPA_AUDIO_FORMAT_U24_LE,
+                    VIRTIO_SND_PCM_FMT_S20 => SPA_AUDIO_FORMAT_S20,
+                    VIRTIO_SND_PCM_FMT_U20 => SPA_AUDIO_FORMAT_U20,
+                    VIRTIO_SND_PCM_FMT_S24 => SPA_AUDIO_FORMAT_S24,
+                    VIRTIO_SND_PCM_FMT_U24 => SPA_AUDIO_FORMAT_U24,
+                    VIRTIO_SND_PCM_FMT_S32 => SPA_AUDIO_FORMAT_S32,
+                    VIRTIO_SND_PCM_FMT_U32 => SPA_AUDIO_FORMAT_U32,
+                    VIRTIO_SND_PCM_FMT_FLOAT => SPA_AUDIO_FORMAT_F32,
+                    VIRTIO_SND_PCM_FMT_FLOAT64 => SPA_AUDIO_FORMAT_F64,
+                    _ => SPA_AUDIO_FORMAT_UNKNOWN,
+                },
+                rate: match params.rate {
+                    VIRTIO_SND_PCM_RATE_5512 => 5512,
+                    VIRTIO_SND_PCM_RATE_8000 => 8000,
+                    VIRTIO_SND_PCM_RATE_11025 => 11025,
+                    VIRTIO_SND_PCM_RATE_16000 => 16000,
+                    VIRTIO_SND_PCM_RATE_22050 => 22050,
+                    VIRTIO_SND_PCM_RATE_32000 => 32000,
+                    VIRTIO_SND_PCM_RATE_44100 => 44100,
+                    VIRTIO_SND_PCM_RATE_48000 => 48000,
+                    VIRTIO_SND_PCM_RATE_64000 => 64000,
+                    VIRTIO_SND_PCM_RATE_88200 => 88200,
+                    VIRTIO_SND_PCM_RATE_96000 => 96000,
+                    VIRTIO_SND_PCM_RATE_176400 => 176400,
+                    VIRTIO_SND_PCM_RATE_192000 => 192000,
+                    VIRTIO_SND_PCM_RATE_384000 => 384000,
+                    _ => 44100,
+                },
+                flags: 0,
+                channels: params.channels as u32,
+                position: pos,
+            };
+
+            let mut audio_info = spa::param::audio::AudioInfoRaw::new();
+            audio_info.set_format(spa::param::audio::AudioFormat::S16LE);
+            audio_info.set_rate(info.rate);
+            audio_info.set_channels(info.channels);
+
+            let values: Vec<u8> = pw::spa::pod::serialize::PodSerializer::serialize(
```

Comment:
```suggestion
            let values: Vec<u8> = spa::pod::serialize::PodSerializer::serialize(
```
to be consistent with the lines above & similarly elsewhere